Dear/Dear/Controllers/NursePrescriptionViewController.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  NursePrescriptionView.swift
+//  Dear
+//
+//  Created by Jeon Jimin on 2022/07/28.
+//
+
+import UIKit
+
+class NursePrescriptionViewController: UIViewController {
```

Comment:
더이상 상속되지 않는 class임을 알 수 있도록 final이 있으면 좋을 것 같아요 ~
```suggestion
final class NursePrescriptionViewController: UIViewController {
```

Reply:
반영해서 수정했습니다!! 감사합니다~!!